src/views/datasets/plankton/summary.vue

```diff
@@ -2,7 +2,8 @@
   <BuoyDashboard :scenarios="scenarios">
     <template #available-data>
       Narragansett Bay Long-Term Plankton Time Series is one of the world’s
-      longest-running plankton surveys. Beginning in 1957, weekly samples have
+      longest-running plankton surveys maintained by the University of
+      Rhode Island. Since 1957, weekly samples have
```

Comment:
I can suggest for silly github reasons, but maybe `The Narragansett Bay Long-Term Plankton Time Series, mainted by the University of Rhode Island, is one of the world's longest-running plankton surveys.`

The current phrasing makes it sounds like it's the longest running survey at uri as opposed to the world